sig-scalability/thresholds.md

```diff
@@ -0,0 +1,96 @@
+# Kubernetes Scalability thresholds
+
+## Background
+
+Since 1.6 release Kubernetes officially supports 5000-node clusters. However,
+the question is what that actually means. As of early Q3 2017 we are in the
+process of defining set of performance-related SLIs ([Service Level Indicators])
+and SLOs ([Service Level Objectives]).
+
+However, no matter what SLIs and SLOs we have, there will always be some users
+coming and saying that their cluster is not meeting the SLOs. And in most cases
+it appears that the reason behind is that we (as developers) have silently
+assumed something (e.g. there will be no more than 10000 services in the
+cluster) and users were not aware of that.
+
+This document is trying to explicitly summarize all the limitations that we are
```

Comment:
limits for number of objects in the system (there's a 'churn' limit that we don't touch here)